storage/badger/operation/chunkDataPacks.go

```diff
@@ -17,6 +17,11 @@ func BatchInsertChunkDataPack(c *badgermodel.StoredChunkDataPack) func(batch *ba
 	return batchWrite(makePrefix(codeChunkDataPack, c.ChunkID), c)
 }
 
+// BatchRemoveChunkDataPack removes a chunk data pack keyed by chunk ID in a batch
+func BatchRemoveChunkDataPack(chunkID flow.Identifier) func(batch *badger.WriteBatch) error {
```

Comment:
Would it be possible to add a comment on expected error returns? I know it is complicated because we return a function ... maybe we could write something like "the returned functions errors with xyz in case ..."